packages/calling/calling/src/CallingClient/line/index.ts

```diff
@@ -0,0 +1,463 @@
+import {Mutex} from 'async-mutex';
+import {
+  ALLOWED_SERVICES,
+  HTTP_METHODS,
+  IDeviceInfo,
+  IpInfo,
+  MobiusDeviceId,
+  MobiusServers,
+  MobiusStatus,
+  RegionInfo,
+  ServiceIndicator,
+  WebexRequestPayload,
+} from '../../common/types';
+import {ICallSettingResponse, ILine, LINE_EVENTS, LineEventTypes, LineStatus} from './types';
+import {
+  LINE_FILE,
+  CALLING_USER_AGENT,
+  CISCO_DEVICE_URL,
+  DISCOVERY_URL,
+  GET_MOBIUS_SERVERS_UTIL,
+  IP_ENDPOINT,
+  KEEPALIVE_UTIL,
+  SPARK_USER_AGENT,
+  URL_ENDPOINT,
+  REGISTRATION_FILE,
+  NETWORK_FLAP_TIMEOUT,
+} from '../constants';
+import log from '../../Logger';
+import {IRegistration} from '../registration/types';
+import {createRegistration} from '../registration';
+import {ClientRegionInfo, ISDKConnector, WebexSDK} from '../../SDKConnector/types';
+import {CallingClientConfig} from '../types';
+import {Eventing} from '../../Events/impl';
+import {LineError} from '../../Errors/catalog/LineError';
+import {METRIC_EVENT, REG_ACTION, METRIC_TYPE} from '../metrics/types';
+import {LOGGER, LogContext} from '../../Logger/types';
+import {handleRegistrationErrors, filterMobiusUris, validateServiceData} from '../../common';
+import SDKConnector from '../../SDKConnector';
+import {ICallManager} from '../calling/types';
+
+export default class Line extends Eventing<LineEventTypes> implements ILine {
+  #webex: WebexSDK;
+
+  #mutex: Mutex;
```

Comment:
Why are we using # instead of saying private here? Any particular reason for the same?

Reply:
awesome thanks did not know that